src/sage/rings/number_field/number_field.py

```diff
@@ -8781,39 +8789,29 @@ def subfields(self, degree=0, name=None):
-                Defn: a1 |--> a^2 + 3/2,
-              None),
-             (Number Field in a2 with defining polynomial x^2 + 4,
-              Ring morphism:
-                From: Number Field in a2 with defining polynomial x^2 + 4
-                To:   Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
-                Defn: a2 |--> 2*a^3 + 7*a,
-              None),
-             (Number Field in a3 with defining polynomial x^2 + 2,
-              Ring morphism:
-                From: Number Field in a3 with defining polynomial x^2 + 2
-                To:   Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
-                Defn: a3 |--> 2*a^3 + 5*a,
-              None),
-             (Number Field in a4 with defining polynomial x^4 + 1,
-              Ring morphism:
-                From: Number Field in a4 with defining polynomial x^4 + 1
-                To:   Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
-                Defn: a4 |--> a^3 + 1/2*a^2 + 5/2*a + 3/4,
-              Ring morphism:
-                From: Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
-                To:   Number Field in a4 with defining polynomial x^4 + 1
-                Defn: a |--> -1/2*a4^3 + a4^2 - 1/2*a4)]
+            [(Number Field in a0 with defining polynomial x, Ring morphism:
+              From: Number Field in a0 with defining polynomial x
+              To:   Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
+              Defn: 0 |--> 0, None),
+            (Number Field in a1 with defining polynomial x^2 - 2, Ring morphism:
+              From: Number Field in a1 with defining polynomial x^2 - 2
+              To:   Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
+              Defn: a1 |--> -a^2 - 3/2, None),
+            (Number Field in a2 with defining polynomial x^2 + 4, Ring morphism:
+              From: Number Field in a2 with defining polynomial x^2 + 4
+              To:   Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
+              Defn: a2 |--> 2*a^3 + 7*a, None),
+            (Number Field in a3 with defining polynomial x^2 + 2, Ring morphism:
+              From: Number Field in a3 with defining polynomial x^2 + 2
+              To:   Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
+              Defn: a3 |--> -2*a^3 - 5*a, None),
+            (Number Field in a4 with defining polynomial x^4 + 1, Ring morphism:
+              From: Number Field in a4 with defining polynomial x^4 + 1
+              To:   Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
+              Defn: a4 |--> -a^3 - 1/2*a^2 - 5/2*a - 3/4, Ring morphism:
+              From: Number Field in a with defining polynomial 2*x^4 + 6*x^2 + 1/2
+              To:   Number Field in a4 with defining polynomial x^4 + 1
+              Defn: a |--> 1/2*a4^3 + a4^2 + 1/2*a4)]
```

Comment:
Something here looks like a messed up merge. This block was changed in #39027, you seem to be reverting some of those changes.

Reply:
Thanks, fixed. Looks like the changes in #39027 are transparent to doctesting.